File: Modal/src/app/model/competencia.ts
export class Competencia {

  private _nivelCompetencia: string;
  private _descripcionCompetencia: string;


  constructor(nivelCompetencia: string, descripcionCompetencia: string) {
    this._nivelCompetencia = nivelCompetencia;
    this._descripcionCompetencia = descripcionCompetencia;
  }


  get nivelCompetencia(): string {
    return this._nivelCompetencia;
  }

  set nivelCompetencia(value: string) {
    this._nivelCompetencia = value;
  }

  get descripcionCompetencia(): string {
    return this._descripcionCompetencia;
  }

  set descripcionCompetencia(value: string) {
    this._descripcionCompetencia = value;
  }
}

File: gs-rest-service-cors-master/complete/src/main/java/com/candidato/entity/Competencia.java
package com.candidato.entity;

/**
 * Clase que representa una competencia de un candidato.
 * @author alejandropalacio
 *
 */
public class Competencia {

	/** Nivel de la competencia del candidato.*/
	private String nivelCompetencia;
	
	/** Descripcion de la competencia. */
	private String descripcionCompetencia;
	
	/**
	 * Constructor de la clase.
	 * @param nivelCompetencia nivel de la competencia.
	 * @param descripcionCompetencia descripcion de la competencia.
	 */
	public Competencia(String nivelCompetencia, String descripcionCompetencia) {
		super();
		this.nivelCompetencia = nivelCompetencia;
		this.descripcionCompetencia = descripcionCompetencia;
	}

	/**
	 * Constuctor por Defecto de la clase.
	 */
	public Competencia() {}
	
	/**
	 * @return the nivelCompetencia
	 */
	public String getNivelCompetencia() {
		return nivelCompetencia;
	}

	/**
	 * @param nivelCompetencia the nivelCompetencia to set
	 */
	public void setNivelCompetencia(String nivelCompetencia) {
		this.nivelCompetencia = nivelCompetencia;
	}

	/**
	 * @return the descripcionCompetencia
	 */
	public String getDescripcionCompetencia() {
		return descripcionCompetencia;
	}

	/**
	 * @param descripcionCompetencia the descripcionCompetencia to set
	 */
	public void setDescripcionCompetencia(String descripcionCompetencia) {
		this.descripcionCompetencia = descripcionCompetencia;
	}
	
	
}
